app/src/app/pages/plando-page/plando-items/plando-items.component.ts

```diff
@@ -0,0 +1,1218 @@
+      { name: "Residental District - Shop Item 5", type: CheckType.SHOP },
+      { name: "Residental District - Shop Item 6", type: CheckType.SHOP },
+      { name: "Port District - Poet Gift", type: CheckType.NORMAL },
+      { name: "Port District - Poet Melody Reward", type: CheckType.NORMAL },
+      { name: "Port District - Fishmael Letter Reward", type: CheckType.NORMAL },
+      { name: "Port District - Radio Trade Event 3 Reward", type: CheckType.NORMAL },
+      { name: "Port District - Hidden Panel", type: CheckType.HIDDEN_PANEL },
+      { name: "Port District - In MultiCoinBlock", type: CheckType.MULTICOIN_BLOCK },
+      { name: "Ruined Castle Grounds - Muss T. Letter Reward", type: CheckType.NORMAL },
+    ]
+  },
+  {
+    name: "Toad Town Tunnels", checks: [
+      { name: "Hall to Blooper 1 (B1) - Hidden Block", type: CheckType.HIDDEN_BLOCK },
+      { name: "Hall to Blooper 1 (B1) - In MultiCoinBlock", type: CheckType.MULTICOIN_BLOCK },
+      { name: "Blooper Boss 1 (B1) - Blooper Fight Reward", type: CheckType.NORMAL },
+      { name: "Short Elevator Room (B1) - Yellow Block Center", type: CheckType.NORMAL },
+      { name: "Short Elevator Room (B1) - Yellow Block Left", type: CheckType.NORMAL },
+      { name: "Short Elevator Room (B1) - Yellow Block Right", type: CheckType.NORMAL },
+      { name: "Spring Room (B2) - Chest On Ledge", type: CheckType.NORMAL },
+      { name: "Elevator Attic Room (B2) - On Parakarry Ledge", type: CheckType.NORMAL },
+      { name: "Elevator Attic Room (B2) - In SuperBlock", type: CheckType.SUPER_BLOCK },
+      { name: "Metal Block Room (B3) - In SuperBlock", type: CheckType.SUPER_BLOCK },
+      { name: "Blue Pushblock Room (B2) - Hidden Block Left", type: CheckType.HIDDEN_BLOCK },
+      { name: "Blue Pushblock Room (B2) - Hidden Block Center", type: CheckType.HIDDEN_BLOCK },
+      { name: "Blue Pushblock Room (B2) - Hidden Block Right", type: CheckType.HIDDEN_BLOCK },
+      { name: "Blue Pushblock Room (B2) - In SuperBlock", type: CheckType.SUPER_BLOCK },
+      { name: "Room with Spikes (B2) - Yellow Block", type: CheckType.NORMAL },
+      { name: "Winding Path (Spiny Room) - Hidden Block Center", type: CheckType.HIDDEN_BLOCK },
+      { name: "Winding Path (Spiny Room) - Hidden Block Right", type: CheckType.HIDDEN_BLOCK },
+      { name: "Winding Path (Spiny Room) - Hidden Block Left", type: CheckType.HIDDEN_BLOCK },
+      { name: "Winding Path (Spiny Room) - Yellow Block", type: CheckType.NORMAL },
+      { name: "Hall to Ultra Boots (B3) - Hidden Block", type: CheckType.HIDDEN_BLOCK },
+      { name: "Hall to Ultra Boots (B3) - Yellow Block Left", type: CheckType.NORMAL },
+      { name: "Hall to Ultra Boots (B3) - Yellow Block Right", type: CheckType.NORMAL },
+      { name: "Ultra Boots Room (B3) - In Big Chest", type: CheckType.NORMAL },
+      { name: "Bridge to Shiver City (B2) - Yellow Block 1", type: CheckType.NORMAL },
```

Comment:
These are all coin blocks. But also, aside from the first one, they are hidden blocks, idk how we should handle this. Is the Hidden block check type really necessary at all?